VideoLocker/res/values/strings.xml

```diff
@@ -165,6 +165,17 @@
     <string name="end_user_title">End User License Agreement</string>
 
     <!-- My Courses -->
+    <!-- Title of course dashboard screen -->
+    <string name="course_dashboard">Dashboard</string>
+    <!-- Course Handout in dashboard -->
+    <string name="course_db_handout">Handouts</string>
+    <!-- Course announcement in dashboard -->
+    <string name="course_db_announcement">Announcement</string>
```

Comment:
Since these aren't the strings for the final design should we separate them out? For example, we don't need them translated. (Also it should be "Announcements" and "Discussions" plural).